nvdaHelper/remote/vbufRemote.cpp

```diff
@@ -152,7 +152,8 @@ int VBufRemote_getTextLength(VBufRemote_bufferHandle_t buffer) {
 int VBufRemote_getTextInRange(VBufRemote_bufferHandle_t buffer, int startOffset, int endOffset, wchar_t** text, boolean useMarkup) {
 	VBufBackend_t* backend=(VBufBackend_t*)buffer;
 	backend->lock.acquire();
-	 wstring textString=backend->getTextInRange(startOffset,endOffset,useMarkup!=false);
+	 wstring textString;
+	 backend->getTextInRange(startOffset,endOffset,textString,useMarkup!=false);
 	backend->lock.release();
 	if(textString.empty()) {

```

Comment:
`backend->getTextInRange` differentiates between "error" and "empty", but this function looks at "empty" to determine whether to return failure. Effectively, I think it amounts to the same thing, but it'd be good to be consistent, unless there's some reason you're deliberately checking empty instead of false here.